tests/MongoDB.Driver.Core.Tests/Core/Bindings/CoreSessionTests.cs

```diff
@@ -217,6 +218,19 @@ public void AdvanceOperationTime_should_have_expected_result()
             Mock.Get(subject.ServerSession).Verify(m => m.Dispose(), Times.Once);
         }
 
+        [Fact]
+        public void StartTransaction_should_throw_when_write_concern_is_unacknowledged()
```

Comment:
Maybe there should probably be a JSON driven test for this?

Reply:
Yeah.  Open a SPEC ticket if there's not one already.